ui/pages/first-time-flow/welcome/index.scss

```diff
@@ -27,8 +27,15 @@
   &__description {
     text-align: center;
 
-    div {
+    p {
       @include Paragraph;
+
+      text-align: start;
+      margin-bottom: 22px;
+    }
+
+    a {
+      color: $Blue-500;
```

Comment:
$primary-1 😬 $Blue-500 is deprecated and i'm a slacker who hasn't furthered that plan still